libs/SalesforceSDK/src/com/salesforce/androidsdk/app/SalesforceSDKManager.java

```diff
@@ -164,6 +165,26 @@ public static boolean hasInstance() {
     }
 
     /**
+     * Sets the app name to be used by the analytics framework.
+     *
+     * @param appName App name.
+     */
+    public static void setAiltnAppName(String appName) {
```

Comment:
The getter, setter and the variable itself have to be static, because the analytics framework initializes this very early in the lifecycle and does it only once per app.